### ISSUE TITLE
Improve loading speed of reports

### ISSUE THREAD

**Issue by apeters** [archesproject/arches#2678]
Some anecdotal test results of page loading (from DISCO dataset):
7.5 sec: http://localhost:8000/report/234f124c-1e17-11e7-b599-6c4008b05c4c
6.8 sec: http://localhost:8000/report/10ccf68e-1b7f-43d8-818c-9889a0a92365
12 sec: http://localhost:8000/report/16baa97e-2c90-4972-ae87-8c7324606867

most of these reports have very little data on them

**Comment by chiatt** [comment]
I'm not sure if this suggestion will apply to report views, but by default the JSONSerializer sorts each serialized object by it's key value.  This is needed in many cases, but not all. If sorting is not necessary, you can pass in the `sort_keys` argument as `False` to speed up serialization.  e.g.`JSONSerializer().serialize(mobile_surveys, sort_keys=False)`

**Comment by chiatt** [comment]
@apeters I made some changes to reduce the number of queries executed when loading reports and added the exclude argument to the JSONSerializer.serialize method to limit what gets serialized in both proxy and regular models.  Only one of the reports listed in the ticket was available when I loaded DISCO: http://localhost:8000/report/234f124c-1e17-11e7-b599-6c4008b05c4c. In that report I reduced the number of queries from: 2854 down to 478 and the DOMContentLoaded time from 5.4 seconds down to 2.63.  Those improvements seem pretty representative of other reports I looked at. I didn't make any changes to the base view. However, it would make sense to do that for #2682 because the map is never loaded in the graph manager page.
